fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -26,6 +26,7 @@
 from fairscale.utils.params import Workhandle, get_global_rank
 
 
+# FIXME? `util/tensor/grad.py`?
```

Comment:
The fate of sharded_ddp should be first determined. We should see if we still need it because it provides something unique that FSDP doesn't already have. For OSS, I think we should keep it since it shard optimizer state differently from FSDP. I am less clear to SDP. cc @blefaudeux